configs/spacy.json

```diff
@@ -11,12 +11,7 @@
     "https://spacy.io/sitemap.xml"
   ],
   "selectors": {
-    "lvl0": {
-      "selector": "//*[contains(@class,'is-active')]/preceding::*[contains(@class,'h0')][1]",
-      "type": "xpath",
-      "global": true,
-      "default_value": "Documentation"
-    },
+    "lvl0": ".h0",
```

Comment:
@ines Why did you remove this? Your current `.h0` element is not only the active element. I will reset the XPATH I have defined

Reply:
My bad, I needed to hard refresh. Make sense